ravenframework/Optimizers/RavenSampled.py

```diff
@@ -715,8 +717,8 @@ def _updateSolutionExport(self, traj, rlz, acceptable, rejectReason):
                      'rejectReason': rejectReason
                     })
     # optimal point input and output spaces
-    if type(self._objectiveVar) == str: # Single Objective Optimization
-      objValue = rlz[self._objectiveVar]
+    if len(self._objectiveVar) == 1: # Single Objective Optimization
+      objValue = rlz[self._objectiveVar[0]]
```

Comment:
I think you need to add an 'else' condition for multi-objective opt.

Reply:
there is one in line 725. Isn't that what you meant?

Reply:
I see, that is not show in the diff.